.github/workflows/build.yaml

```diff
@@ -134,6 +134,12 @@ jobs:
 
       - run: ./gradlew -p samples/emoji-search build
 
+      - uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
+          emulator-boot-timeout: 20000
```

Comment:
Have we ever really figured out why the emulator boot would timeout in SQLDelight PR checks?
Hopefully this doesn't introduce flakiness.

Reply:
We have two other jobs that already have emulators. They don't fail that often. Maybe one out of every 30 PRs. Not great, but not terrible either.